data.php

```diff
@@ -5,13 +5,7 @@
 
 $files = scandir("data");
 
-$dimensions = array(
-    "Culture and Org." => readYaml("data/CultureandOrg.yml"),
-    "Build and Deployment" => readYaml("data/BuildandDeployment.yml#/sub-dimensions"),
-    "Information Gathering" => readYaml("data/Informationgathering.yml"),
-    "Implementation" => readYaml("data/Implementation.yml"),
-    "Test and Verification" => readYaml("data/TestandVerification.yml")
-);
+$dimensions = readYaml("data/dimensions.yaml");
```

Comment:
Dimensions are now inherited directly from `dimensions.yaml`. No need to modify the php every time.